src/common/k8s-api/endpoints/helm-releases.api/get-configuration.injectable.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { urlBuilderFor } from "../../../utils/buildUrl";
+import { apiBaseInjectionToken } from "../../api-base";
+
+export type RequestHelmReleaseConfiguration = (
+  name: string,
+  namespace: string,
+  all: boolean
+) => Promise<string>;
+
+const requestConfigurationEnpoint = urlBuilderFor("/v2/releases/:namespace/:name/values");
+
+const requestHelmReleaseConfigurationInjectable = getInjectable({
+  id: "call-for-helm-release-configuration",
+
+  instantiate: (di): RequestHelmReleaseConfiguration => {
+    const apiBase = di.inject(apiBaseInjectionToken);
+
+    return (name, namespace, all: boolean) => (
+      apiBase.get(requestConfigurationEnpoint.compile({ name, namespace }, { all }))
+    );
+  },
+
+  causesSideEffects: true,
```

Comment:
Hmm. Shouldn't be so for this injectable? Maybe for `apiBase`?

Reply:
Done